src/App.js

```diff
@@ -121,36 +134,84 @@ export default class App extends Component {
     );
   }
 
+  /**
+   * Callback function to update "viewport" in state
+   * @param newViewport 
+   */
+  updateViewport = (newViewport) => {
+    const {viewport} = this.state;
+    this.setState({
+      viewport: {
+        ...viewport,
+        ...newViewport,
+      }
+    });
+  }
+
+  /**
+   * Returns a list of the county GeoJSON features
+   */
+  getIllinoisCountyFeatures = () => {
+    if (this.state.illinois_counties != null) {
+      var countyFeatures =  this.state.illinois_counties.features.map((feature) => {
```

Comment:
//just condensing this to one line!
const countyFeatures =  this.state.illinois_counties.features.map((feature) => feature)

Reply:
Changed